editor/effects.js

```diff
@@ -2,12 +2,12 @@
  * External dependencies
  */
 import { get } from 'lodash';
-import { parse, stringify } from 'querystring';
 
 /**
- * WordPress dependencies
+ * Internal dependencies
  */
 import { getBlockSettings, switchToBlockType } from 'blocks';
```

Comment:
There's both WordPress and Internal dependencies here.

Aside: We could do for a better indicator.